### PR TITLE
access kolibri using docker new blog

### PR DESCRIPTION
**Pull request by irisb1701**
https://raw.githack.com/treehouses/treehouses.github.io/kolibri/#!./pages/blog/20191014-kolibri.md

### CODE REVIEW

File: pages/blog/20191014-kolibri.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Kolibri is a platform used by our "competitor" organization called Learning Equality similar to what Planet is for Open Learning Exchange.
+
+Below are instructions on how to run Kolibri on treehouses:
+
+1. Plug your RPi in and ssh into treehouses
+1. If you don't already have docker installed, follow [these instructions](https://docs.docker.com/v17.09/engine/installation/) on their official page to install it on mac/windows
+1. Now, go to the [treehouses github](https://github.com/treehouses)
+1. Scroll down until you find the `kolibri` repository
+1. Once in the repo, scroll down to where the readme.md file is displayed
+1. Copy the link provided: `docker run -v /data/save/location:/root/.kolibri -p 8080:8080 treehouses/kolibri
```
Comment (XavierElon) [inline pages/blog/20191014-kolibri.md]: You forgot a tick here to close the coding statement

Comment (CalebProvost) [inline pages/blog/20191014-kolibri.md]: > Copy the link provided: `docker run -v /data/save/location:/root/.kolibri -p 8080:8080 treehouses/kolibri

forgot the ending `

File: pages/blog/20191014-kolibri.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+1. Replace /data/save/location with /srv/kolibri
```
Comment (CalebProvost) [inline pages/blog/20191014-kolibri.md]: > Replace /data/save/location with /srv/kolibri

Should the directories be in code blocks?
"Replace `/data/save/location` with `/srv/kolibri`"

**Review by CalebProvost — CHANGES_REQUESTED** [review]

**Review by CalebProvost — APPROVED** [review]

**Review by XavierElon — APPROVED** [review]
Worked for me

**Review by dogi — APPROVED** [review]

**Pull request merged**